Description: Don't build the osgframerenderer example if building for
 OpenGL ES 1, OpenGL ES 2, or OpenGL 3. This is a portion of the full commit.
Origin: upstream, https://github.com/openscenegraph/osg/commit/3f812e42c7bb257e1e19604318becd359d21f469

Index: openscenegraph/OpenSceneGraph/examples/CMakeLists.txt
===================================================================
--- openscenegraph.orig/OpenSceneGraph/examples/CMakeLists.txt	2014-07-26 08:49:44.583787000 -0500
+++ openscenegraph/OpenSceneGraph/examples/CMakeLists.txt	2014-08-03 11:41:48.439983802 -0500
@@ -46,7 +46,6 @@
     ADD_SUBDIRECTORY(osgfont)
     ADD_SUBDIRECTORY(osgforest)
     ADD_SUBDIRECTORY(osgfxbrowser)
-    ADD_SUBDIRECTORY(osgframerenderer)
     ADD_SUBDIRECTORY(osgoutline)
     ADD_SUBDIRECTORY(osggameoflife)
     ADD_SUBDIRECTORY(osggeometry)
@@ -173,6 +172,7 @@
 
     IF(NOT OSG_GLES1_AVAILABLE AND NOT OSG_GLES2_AVAILABLE AND NOT OSG_GL3_AVAILABLE)
         ADD_SUBDIRECTORY(osgscreencapture)
+        ADD_SUBDIRECTORY(osgframerenderer)
         ADD_SUBDIRECTORY(osgmotionblur)
         ADD_SUBDIRECTORY(osgteapot)
     ENDIF()
